Client/src/test/java/org/projectcardboard/client/controller/SoundEffectPlayerTest.java

```diff
@@ -0,0 +1,17 @@
+package org.projectcardboard.client.controller;
+
+import org.junit.Test;
+import org.projectcardboard.client.controller.SoundEffectPlayer.SoundName;
+
+public class SoundEffectPlayerTest {
+
+    @Test
+    public void staticMapLoaded() {
+        /**
+         * Because this is static, private, void the only way to 
```

Comment:
Since static things cannot be unit-tested/mocked easily... how hard would it be to turn the player into something not static?

Reply:
I'm not sure, but I do know I don't want to do it as part of this PR.